wandb/sdk/wandb_settings.py

```diff
@@ -318,11 +318,11 @@ class SettingsData:
     _file_stream_retry_wait_min_seconds: int  # min wait time between retries
     _file_stream_retry_wait_max_seconds: int  # max wait time between retries
     _file_stream_timeout_seconds: int  # timeout for individual HTTP requests
-    # file uploader retry client configuration
-    _file_uploader_retry_max: int
-    _file_uploader_retry_wait_min_seconds: int
-    _file_uploader_retry_wait_max_seconds: int
-    _file_uploader_timeout_seconds: int
+    # file transfer retry client configuration
+    _file_transfer_retry_max: int
+    _file_transfer_retry_wait_min_seconds: int
+    _file_transfer_retry_wait_max_seconds: int
+    _file_transfer_timeout_seconds: int
```

Comment:
Note: this is a breaking change, but it's probably okay at this stage.

Reply:
cc. @kptkin